.github/workflows/validate-this-gha-refs-pr.yml

```diff
@@ -0,0 +1,76 @@
+name: Validate GHA Refs - Pull Request
+on:
+  pull_request:
+  
+jobs:
+  check-pr-reviews:
+    runs-on: ubuntu-22.04
+    permissions:
+      actions: read
+      contents: read
+      pull-requests: read
+    outputs:
+      approved-count: ${{ steps.check-pr-reviews.outputs.approved-count }}
+      request-changes-count: ${{ steps.check-pr-reviews.outputs.request-changes-count }}
+      comment-count: ${{ steps.check-pr-reviews.outputs.comment-count }}
```

Comment:
Do we need these? I think they are not used (also in the other places).

Reply:
Good catch - no, not anymore. I originally (and incorrectly) assumed that Request for Changes reviews would change to Approved reviews when accepted, and the `if:` condition was `approved > 0 && changes == 0`. But instead I needed to look at the most recent review to determine if it was approved. I'll remove these and resubmit.